ViAn/Filehandler/analysis.h

```diff
@@ -0,0 +1,58 @@
+#ifndef ANALYSIS_H
+#define ANALYSIS_H
+#include <QJsonObject>
+#include <QJsonArray>
+#include <QString>
+#include <QObject>
+#include <vector>
+#include <map>
+#include "saveable.h"
+#include "opencv2/core/core.hpp"
+
+enum ANALYSIS_TYPE {MOTION_DETECTION = 0, FACIAL_DETECTION = 1};
+class OOI : Saveable{
+    int frame;
+    std::string file_path;
+    std::pair<int, int> upper_left;
+    std::pair<int, int> lower_right;
```

Comment:
members or functions declared within a class is default private. So they are private